client/state/reader/lists/test/actions.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import nock from 'nock';
+import sinon from 'sinon';
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	READER_LISTS_RECEIVE,
+	READER_LISTS_REQUEST
+} from 'state/action-types';
+import {
+	receiveLists,
+	requestSubscribedLists
+} from '../actions';
+
+describe( 'actions', () => {
+	const spy = sinon.spy();
+
+	beforeEach( () => {
+		spy.reset();
+	} );
+
+	describe( '#receiveLists()', () => {
+		it( 'should return an action object', () => {
+			const lists = [ { ID: 841, title: 'Hello World', slug: 'hello-world' } ];
+			const action = receiveLists( lists );
+
+			expect( action ).to.eql( {
+				type: READER_LISTS_RECEIVE,
+				lists
+			} );
+		} );
+	} );
+
+	describe( '#requestSubscribedLists()', () => {
+		before( () => {
+			nock( 'https://public-api.wordpress.com:443' )
+				.persist()
+				.get( '/rest/v1.2/read/lists' )
+				.reply( 200, {
+					found: 2,
+					lists: [
+						{ ID: 841, title: 'Hello World' },
+						{ ID: 413, title: 'Mango & Feijoa' }
+					]
+				} );
+		} );
```

Comment:
just did a quick review on how `before` and `after` work. Wouldn't hurt to add a `nock.cleanAll` in an after.